themes/vitesse-dark-soft.json

```diff
@@ -127,7 +127,7 @@
     "panelTitle.activeForeground": "#dbd7caee",
     "panelTitle.inactiveForeground": "#959da5",
     "panelInput.border": "#2f363d",
-    "terminal.foreground": "#dbd7caee",
+    "terminal.foreground": "#dbd7ca",
```

Comment:
You should update the src/theme.ts instead of .json, this fill is generated.

Reply:
I made adjustments